prow/cmd/branchprotector/BUILD.bazel

```diff
@@ -19,8 +19,12 @@ load("@io_bazel_rules_docker//go:image.bzl", "go_image")
 
 go_image(
     name = "image",
-    binary = ":branchprotector",
```

Comment:
Did we stop using binary because it does not allow us to set goarch/goos?

Reply:
I did some testing and reading of the current go image rules implementations, we _could_ use a go binary but if we want to set goarch/goos we need a second binary target (so you can still build and run these on other platforms to locally test). 
This is also preferred by the go_image rules and is more concise. /shrug